docs/adding-new-dapps.md

```diff
@@ -17,7 +17,8 @@ There are some fields that need to be translated into different languages, and t
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `name`            | The name of the Dapp.                                                                                                                                                                                                                                     |
 | `id`              | A unique identifier for the Dapp. The current pattern is to use a lower-kebab-case string.                                                                                                                                                                |
-| `categoryId`      | The category id under which the Dapp should show up.                                                                                                                                                                                                      |
+| `categoryId`      | The category id under which the Dapp should show up. This will be deprecated soon, it will be replaced by `categories`.                                                                                                                                   |
```

Comment:
I propose we directly remove `categoryId` here and update the Cloud Function to return the right shape based on the Valora version.
The first one would be used for existing clients.

Reply:
Yes, but what do we do with the apps that are currently in `Lend, Borrow & Earn`?
We have to split that category, and I didn't want to change the current behavior.